apps/web/src/lib/core/component/AI/component/tool/calendar/EventPreview.css

```
@@ -0,0 +1,77 @@
+.calendar-tool-preview .calendar-view .fc .calendar-event-emphasized {
+	--fc-event-bg-color: color-mix(
+		in oklch,
+		var(--event-calendar-color, var(--color-calendar)) 22%,
+		var(--color-surface)
+	);
+	--fc-event-text-color: color-mix(
+		in oklch,
+		var(--event-calendar-color, var(--color-calendar)) 72%,
+		var(--color-ink)
+	);
+	z-index: 4;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)